warehouse/oso_mcp/oso_mcp/utils/entity_context.py

```diff
@@ -1,235 +1,50 @@
 import json
+import re
 from typing import Literal
 
+import sqlglot as sg
 from oso_agent.util.config import AgentConfig
 from oso_agent.workflows.entity_context.llm_structured_output_workflow import (
     create_llm_structured_output_workflow,
 )
 from pydantic import BaseModel, Field
+from sqlglot import expressions as exp
 
 
 class EntityVariantsOutput(BaseModel):
     variants: list[str] = Field(..., description="Possible variants for the entity name.")
 
 class EntitySelectionOutput(BaseModel):
-    selected: list[str] = Field(..., description="Selected canonical entity names.")
+    selected: str = Field(..., description="Selected canonical entity name.")
```

Comment:
```suggestion
    selected: str = Field(description="Selected canonical entity name.")
```

I don't think the `...` is necessary. Mostly a nit, but it's unnecessary bits of text.